### PR TITLE
Add User Details Disable/Delete tab

### PR DESCRIPTION
**Pull request by toml5566**
ref DEV-1752

Screenshot 2024-08-16 at 3 39 03 PM
Screenshot 2024-08-16 at 3 39 10 PM
Screenshot 2024-08-16 at 3 39 22 PM
Screenshot 2024-08-16 at 3 39 32 PM
Screenshot 2024-08-16 at 3 39 39 PM

### CODE REVIEW

File: portal/src/graphql/adminapi/UserDetailsAdminActions.tsx [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+        <Text className={cn(styles.cellLabel, styles.actionCellLabel)}>
+          <FormattedMessage id="UserDetailsAdminActions.disable-user.title" />
+        </Text>
+        <Text className={cn(styles.actionCellBody, styles.actionCellLast)}>
+          <FormattedMessage id="UserDetailsAdminActions.disable-user.body" />
+        </Text>
+        {isDisabled ? (
+          <DefaultButton
+            theme={themes.actionButton}
+            className={cn(styles.actionCellActionButton, styles.actionBorder)}
```
Comment (louischan-oursky) [inline portal/src/graphql/adminapi/UserDetailsAdminActions.tsx:49]: actionCellActionButton modifies padding, which is considered as the implementation details of DefaultButton. You use `styles` to override padding. In general, you use `className` to position the component, and use specific prop, like `variant`, `styles` to customize the Fluent component.

File: portal/src/graphql/adminapi/UserDetailsScreen.tsx [hunk 2]
```diff
@@ -344,6 +347,27 @@ const UserDetails: React.VFC<UserDetailsProps> = function UserDetails(
...
           <FormattedMessage id="UserDetailsScreen.user-anonymized.message" />
         </MessageBar>
+        <Pivot
+          styles={{
+            itemContainer: {
+              flex: "1 0 auto",
+              display: "flex",
+              flexDirection: "column",
+            },
+          }}
```
Comment (louischan-oursky) [inline portal/src/graphql/adminapi/UserDetailsScreen.tsx:385]: You define constants with `const` in this file.

**Review by louischan-oursky — CHANGES_REQUESTED** [review]

File: portal/src/graphql/adminapi/UserDetailsAdminActions.tsx [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+            iconProps={{ iconName: "Play" }}
+            text={
+              <FormattedMessage id="UserDetailsAdminActions.disable-user.action.enable" />
+            }
+            onClick={onDisableData}
+          />
+        ) : (
+          <DefaultButton
```
Comment (louischan-oursky) [inline portal/src/graphql/adminapi/UserDetailsAdminActions.tsx:57]: Introduce a new component AdminActionButton. Handle merging the styles from prop, and the inherit styles of the design of this component. useMergedStylesPlain

**Review by louischan-oursky — CHANGES_REQUESTED** [review]

**Comment by toml5566** [comment]
@louischan-oursky Updated, please feel free to review

**Review by louischan-oursky — APPROVED** [review]

**Pull request merged**